Week2/homework/js-exercises/evenOddReporter.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+var evenOdd = []
```

Comment:
Why did you create this array? Because it is not being used

Reply:
erased

Reply:
https://github.com/HackYourFuture/JavaScript1/pull/222/files

i am trying to git push my homework again, but it accrued an errror:

 ! [rejected]        week2-enwer-homework -> week2-enwer-homework (fetch first)
then i did as they recommmended, still did not see the new branch.